WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -115,7 +115,7 @@ - (void)backfillPostsForTopic:(ReaderAbstractTopic *)topic success:(void (^)(NSI
 
 - (void)toggleLikedForPost:(ReaderPost *)post success:(void (^)())success failure:(void (^)(NSError *error))failure
 {
-    [self.managedObjectContext performBlock:^{
+    [self.managedObjectContext performBlockAndWait:^{
```

Comment:
Sorry about being annoying!. `performBlockAndWait` makes me a bit nervous (RE: anything that might backfire into the mainContext _might_ cause a deadlock).

Would it be possible to switch it back?